### PR TITLE
update operator-sdk to use local/bin

### PR DESCRIPTION
**Pull request by yazug**
specify version for operator-sdk for make bundle
download operator-sdk pinned version when generating bundle update make bundle to use local operator-sdk

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -193,11 +193,12 @@ $(LOCALBIN):
 KUSTOMIZE ?= $(LOCALBIN)/kustomize
 CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
-GINKGO ?= $(LOCALBIN)/ginkgo
```
Comment (booxter) [inline Makefile:196]: isn't the GINKGO variable used below `ginkgo` target?

Comment (yazug) [reply Makefile:196]: my bad yea missed that let me fix that up

Comment (yazug) [reply Makefile:196]: done

**Review by karelyatin — APPROVED** [review]

**Pull request merged**